test/integration/controllers/WebActionsController.test.js

```diff
@@ -40,12 +104,12 @@ describe('WebActionsController', () => {
           invitePhone1: null,
           invitePhone2: '',
           invitePhone3: undefined,
-        }
+        },
       };
 
       const data = WebActionsController.createMobileCommonsReferralRequest(mockRequest);
       assert.equal(data, false);
-    })
+    });
 
     it('should handle building the request for 1 friend referral', () => {
```

Comment:
Does this mean that 3 friend referrals are not required to enter into scholarship? (can refer 1 or 2 friends only)